packages/rest-explorer/src/rest-explorer.types.ts

```diff
@@ -29,4 +29,21 @@ export type RestExplorerConfig = {
    * forwarded to the application behind the proxy.
    */
   useSelfHostedSpec?: false;
+
+  /**
+   * Mount custom asset directory to endpoint `/assets` on the server.
+   * It is the relative path to module swagger-ui-dist, which
+   * is under `@loopback/rest-explorer/node_modules`.
+   *
+   * Examples:
+   * path to swagger-ui themes provided in `@loopback/rest-explorer`:
+   * '../../css'
+   * path to assets provided in app:
+   * '../../../../<asset_folder_name>'
+   */
+  assetDir?: string;
+  /**
+   * The custom swagger-ui theme file's path.
+   */
+  swaggerThemePath?: string;
```

Comment:
Oh, I see. The clients are expected to provide `assetDir` to be mounted via static middleware, and then configure `swaggerThemePath` to point to a file inside `assetDir`. Can you please clarify this information in the tsdoc comments and/or the documentation?

IIRC, in loopback-component-explorer, we were mounting the asset directory at `/`, thus allowing the users to overwrite any files provided by swagger-ui. (The important part of the trick is to mount swagger-ui files first and `assetDir` second.) Have you considered using this approach too? If we do that, then I think `swaggerThemePath` is no longer needed, because we can ask users to save their CSS as `swaggar-ui.css` inside `assetDir`. WDYT?

Either way, please add some tests to verify how these new config options are handled.

Reply:
added test and @hacksparrow helps simplified the code, now we only need one config property :)